Simple-step-by-step-setup-instructions-for-non-Linux-users.md

```diff
@@ -0,0 +1,225 @@
+
+The easiest way is to run `graft_server` for a few seconds, and then stopping the process. It will create a stake-wallet automatically.
+
+* **You will need to create a new Terminal** - if you close the terminal window, it will kill the current process, in this case `graftnoded` - in future tutorials we will explain how to use `screen` to solve this, but for now we're keeping it simple.
+
+* So, open a new Terminal session and
+
+
+    cd supernode
+    ./graft_server
+
+* Give is a brief few seconds, 5-10s, and then quit by CTRL+C, to stop the supernode graft_server.
+
+* Let’s open the stake-wallet to find the address to send funds to. By opening the wallet, you will be able to retrieve the mnemonic seed phrase for safe keeping. The actual stake-wallet is created in a hidden folder, here is how to access it;
+
+
+    cd $HOME/supernode/BUILD/bin
+    ./graft-wallet-cli --wallet-file ~/.graft/supernode/data/stake-wallet/stake-wallet --password "" --testnet --trusted-daemon
+
+![16](simple-setup-images/16.png)
+
+* Here you can type `help` too, to see all the command you can perform within the wallet, like for example `show_transfers`.
+
+* Copy the wallet address, make sure it is a **testnet** address and starts with an **F**. Send the amount needed for the tiered node you wish to run, and wait until the balance is unlocked. If you don’t have testnet GRFT, ask ~~the dev team~ in the testnet chat channel on Discord (https://discord.gg/a59J3Z) or Telegram (https://t.me/joinchat/F25OCUXOQ55qqHm_BZU4Sg)
+
+### **Never ever use your mainnet wallet with real mainnet GRFT.**
+<br/>
+
+
+* Check your funds with `balance` and/or `refresh`
+
+![17](simple-setup-images/17.png)
+
+Once your balance is unlocked, you can start the supernode in the next step.
+
+* Exit the wallet (or create a new Terminal session), navigate to the `$HOME/supernode` directory and start `graft_server`
+
+
+    CTRL + C  (or type Exit)
+    cd $HOME/supernode
+    ./graft_server
+
+You should start seeing something like:
+
+![18](simple-setup-images/18.png)
+
+If you check back on the `graftnoded` Terminal, you will see this going on, as `graftnoded` and `graft_server` are now talking to each other;
+
+![19](simple-setup-images/19.png)
+
+It is normal to see these errors. The `graft_server` is processing blocks to get up to speed with the network. Let it run, it is quite intensive. You will start progressively seeing more good announces like this;
+
+![20](simple-setup-images/20.png)
+
+Eventually, after quite some time, it will finish. Let it run. Some terminal emulators may even appear to be stuck or hanging. After some time, RPC Request: on_supernode_announce: start and end should prevail.
+
+![21](simple-setup-images/21.png)
+
+![22](simple-setup-images/22.png)
+
+#### Congratulations!
+You have a fully functioning Supernode live on Alpha 3 testnet, and should start getting rewards soon.
```

Comment:
Perhaps add something here about asking for a testnet stake on telegram; otherwise you just have a "T0"